apps/web/src/routes/UserSettings/NotificationSettingsTab/NotificationSettingsTab.js

```diff
@@ -132,88 +127,95 @@ function NotificationSettingsTab ({
+              update={updateUserSetting('dmNotifications')}
+            />
           </div>
-          <SettingsToggles
-            label={<span className='flex items-center'>Receive group notifications by <InfoButton content='This controls how you receive notifications for all your groups.' /></span>}
-            settings={allGroupsSettings}
-            update={updateAllGroupsAlert}
-          />
-
-          <div className='flex items-center justify-between mt-2'>
-            <span>{t('Send new post notifications in this group for')}</span>
-            <Select
-              value={allGroupsSettings.postNotifications}
-              onValueChange={value => updateAllGroupsAlert({ postNotifications: value })}
-            >
-              <SelectTrigger className='inline-flex w-auto'>
-                <SelectValue />
-              </SelectTrigger>
-              <SelectContent>
-                <SelectItem value='none'>{t('No Posts')}</SelectItem>
-                <SelectItem value='important'>{t('Important Posts (Announcements & Mentions)')}</SelectItem>
-                <SelectItem value='all'>{t('Every Post')}</SelectItem>
-                <SelectItem value='mixed' disabled>{t('~ Mixed ~')}</SelectItem>
-              </SelectContent>
-            </Select>
+          <div className='py-2'>
+            <SettingsToggles
+              label={<span className='text-xl'><Icon name='Messages' className='mr-2' />{t('Comments on followed posts')}</span>}
+              settings={getCurrentSettings(me, 'commentNotifications')}
+              update={updateUserSetting('commentNotifications')}
+            />
           </div>
-          <div className='flex items-center justify-between mt-2'>
-            <span className=''>{t('Send me an email digest for this group')}</span>
-            <Select
-              value={allGroupsSettings.digestFrequency}
-              onValueChange={value => updateAllGroupsAlert({ digestFrequency: value })}
-            >
-              <SelectTrigger className='inline-flex w-auto'>
-                <SelectValue />
-              </SelectTrigger>
-              <SelectContent>
-                <SelectItem value='daily'>{t('Daily')}</SelectItem>
-                <SelectItem value='weekly'>{t('Weekly')}</SelectItem>
-                <SelectItem value='never'>{t('Never')}</SelectItem>
-                <SelectItem value='mixed' disabled>{t('~ Mixed ~')}</SelectItem>
-              </SelectContent>
-            </Select>
+        </div>
+        <div className='text-sm text-foreground/50 mb-2 mt-4'>{t('Default group notifications')}</div>
+        <div className='bg-card/100 rounded-lg p-4 shadow-lg mb-4'>
+          <div>
+            <div className='flex items-center text-sm text-foreground/80 mb-2'>
+              <span>{t('These settings apply to all groups you are a member of. Any group-specific settings will override these defaults for that group.')}</span>
```

Comment:
this is not quite accurate. the way these toggles work is turning it on turns on the setting for all the groups, turning it off turns off the setting for all the groups. if all the groups have this setting on the toggle will show as on, if even one group has it turned off then the toggle will show as off here. that could be a little confusing, ideally we would show it in some sort of half/mixed state if some groups have it on and others off. but there's not a default/override thing happening

Reply:
Ok, I updated the language to try to reflect what I'm understanding here -- what do you think?